client/gutenberg/editor/style.scss

```diff
@@ -177,11 +177,7 @@ $gutenberg-theme-toggle: #11a0d2;
 	}
 
 	// UNSET CALYPSO DEFAULT STYLES
-	input[type='text'],
```

Comment:
Thanks for the update @torres126! Could we please bring this block back to avoid causing other regressions that Calypso default styles can cause. E.g:

<img width="308" alt="screen shot 2018-12-03 at 23 54 05" src="https://user-images.githubusercontent.com/1182160/49406961-5bd98e80-f757-11e8-84d4-772cbbbeb637.png">

(notice the password input field width)

Reply:
Good spot, thanks for pointing that out! I've added it for now, but I'm not sure how effective it'll be. If it's just the password field input width, maybe it should just be more specific? 

```suggestion
	input[type='text'],
  .components-popover input[type="search"].editor-inserter__search {
 	        width: auto;
 	}
 	
 .is-section-gutenberg-editor input[type='text'], .is-section-gutenberg-editor input[type='search'] {
 			width: 100%;
  }
  
  
.editor-post-visibility__dialog-password input[type='text'] {
  width: auto;
  }
```
But that's under the assumption that a change still causes any issues.

Reply:
I think that's probably best, better than running the risk of something causing an issue because of this in the future, or because of something not spotted, as you've said. Let me know if there's anything else you recommend changing though. :)